File: pinyougou-user-web/src/main/webapp/js/service/userService.js
app.service("userService",function ($http) {
    //1.获取验证码
    this.getValidCode=(phone)=>{
        return $http.get("./user/getValidCode.do?phone="+phone);
    }
    //2.用户注册
    this.add = (entity,validCode)=>{
        return $http.post("./user/add.do?validCode="+validCode,entity);
    }
})

File: pinyougou-user-interface/src/main/java/com/pinyougou/user/service/UserService.java
package com.pinyougou.user.service;

import com.pinyougou.pojo.TbUser;

public interface UserService {

        //1.获取手机验证码
        void getValidCode(String phone);
        //2.比较验证是否正确
        boolean checkValidCode(String validCode,String phone);
        //3.添加用户
        void add(TbUser user);

}
